setup.py

```diff
@@ -92,6 +92,8 @@ def build_extension(self, ext):
         "nle-play = nle.scripts.play:main",
         "nle-ttyrec = nle.scripts.ttyrec:main",
         "nle-ttyplay = nle.scripts.ttyplay:main",
+        "nle-ttyplay2 = nle.scripts.ttyplay2:main",
```

Comment:
the ttyplay2 file doesnt have a main function? or something?

I'm getti
<img width="1169" alt="Screenshot 2021-11-02 at 15 10 31" src="https://user-images.githubusercontent.com/6815729/139874584-9b52c8f8-edd7-46d8-8741-006aa0bbb087.png">
ng

Reply:
Good catch; fixed.